scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -89,45 +93,39 @@ We could also use a `for` loop, which is the approach that many programming lang
 A different and more modular coding approach to reading in these files (and more!) is to leverage the [`purrr`](https://purrr.tidyverse.org/) `tidyverse` package, which provides a convenient set of functions for operating on lists.
 You can read more about these functions and their power and utility in R in [Chapter 21 of _R for Data Science_](https://r4ds.had.co.nz/iteration.html#for-loops-vs.-functionals).
 
-Of particular interest is the [`purrr::map()`](https://purrr.tidyverse.org/reference/map.html) family of functions, which can be used to run a given function on each element of a list or vector in one call.
+Of particular interest is the [`purrr::map()`](https://purrr.tidyverse.org/reference/map.html) family of functions, which can be used to run a given function on each element of a list (or vector) in one call.
 The general syntax for `purrr::map()` and friends is:
 
 ```
 # Syntax for using the map function:
-purrr::map(<input vector or list>, 
-           <function to apply to each item in the input>, 
-           <any additional arguments to the function can go here>, 
-           <and also here if there are even more arguments, and so on>)
+purrr::map(input list or vector, 
+           function to apply to each item in the input, 
+           any additional arguments to the function can go here, 
+           and also here if there are even more arguments, and so on)
```

Comment:
I didn't notice this change, but I would probably revert the removal of the angle brackets. Having the bare text seems confusing to me.